spec/Configuration/Dotenv/ParseSpec.hs

```diff
@@ -142,11 +142,26 @@ spec = describe "parse" $ do
   it "doesn't allow more configuration options after a quoted value" $
     parseConfig `shouldFailOn` "foo='bar'baz='buz'"
 
-  it "parses a command $(command)" $ do
-    parseConfig "FOO=$(command)"
-      `shouldParse` [ParsedVariable "FOO" (Unquoted [CommandInterpolation "command"])]
-    parseConfig "FOO=asdf_$(command)"
-      `shouldParse` [ParsedVariable "FOO" (Unquoted [VarLiteral "asdf_", CommandInterpolation "command"])]
+  context "$(command) interpolation" $ do
+    it "parses a simple command" $ do
+      parseConfig "FOO=$(command)"
+        `shouldParse` [ParsedVariable "FOO" (Unquoted [CommandInterpolation "command" []])]
+
+    it "parses a command anywhere in the value" $ do
+      parseConfig "FOO=asdf_$(command)"
+        `shouldParse` [ParsedVariable "FOO" (Unquoted [VarLiteral "asdf_", CommandInterpolation "command" []])]
+
+    it "parses a command with arguments" $ do
+      parseConfig "FOO=$(command arg1 arg2)"
+        `shouldParse` [ParsedVariable "FOO" (Unquoted [CommandInterpolation "command" ["arg1", "arg2"]])]
+
+    it "parses a command with quoted arguments" $ do
+      parseConfig "FOO=$(command \"arg 1\" arg2)"
+        `shouldParse` [ParsedVariable "FOO" (Unquoted [CommandInterpolation "command" ["arg 1", "arg2"]])]
+
+    it "parses a command with arguments separated by newlines" $ do
+      parseConfig "FOO=$(command \"arg 1\"\narg2\n)"
```

Comment:
@pbrisbin Could you please add a couple of tests to cover the cases that you mention in the issue:
1. `parseConfig "FOO=$(foo-bar.sh)"`
2. `parseConfig "FOO=$(foo/bar.sh)"`

?
Thanks!

Reply:
Sure, will do!